docs/variants/workstation/installation-manual.md

```diff
@@ -0,0 +1,218 @@
+   the Master Section of the flash descriptor are NOT in effect. Please note
+   that Protected Range (PR) restrictions still apply.
+   ```
+
+   Newer version of flashrom may not display the warning about unsupported
+   chipset as it already may be marked as tested. Our team has verified that the
+   flashrom updates firmware reliably on this chipset.
+
+## Verification
+
+1. If everything went well (flashrom has verified the flash content),
+2. Shut down machine, move jumper to the original place
+3. Power on the machine.
+4. After rebooting you should see the Dasharo Workstation logo when booting.
+   When logo appears you may press ESC to select the boot device if you want to
+   reboot from other source.
+
+   ![](../../images/dasharo-black.jpg)
+
+## Troubleshooting
+
+If you do not see the logo after few seconds, something probably went wrong or
+a bug was hit. If the LED on the power button shines white, that means the
+platform booted correctly.
+
+![](../../images/white_led.jpg)
+
+If the power button LED shines constantly in orange color, that means you have
+hit an error. The LED will soon to start blinking.
+
+![](../../images/orange_led.jpg)
+
+If you see the logo and after that system do not starts (black screen), please
+take the following steps:
+
+1. Put bootable USB stick to USB port.
+2. Restart computer using power button.
+3. Press ESC key to enter a boot menu.
+4. Choose USB drive from the list.
+5. Re-install operating system.
+
+If you see an flashrom error like this:
+
+``` console
+ERROR: Could not get I/O privileges (Operation not permitted).
+You need to be root.
+Error: Programmer initialization failed.
+```
+
+Use `sudo` with the flashrom command. If the problem persists, probably the
+kernel restricts access to IOMEM. In order to workaround it, append
+`iomem=relaxed` to kernel commandline. You may do it in two ways:
+
+1. Edit `grub.cfg` in `/boot/grub/`:
+  ``` bash
+  linux /boot/vmlinuz-4.15.0-115-generic ro quiet iomem=relaxed`
+  ```
+  And reboot. Then try flashing again.
+
+2. Edit `/etc/default/grub`:
+  ``` bash
+  GRUB_CMDLINE_LINUX="iomem=relaxed"`
+  ```
+  And regenerate grub config file with `sudo update-grub2` or
+  `sudo grub-mkconfig -o /boot/grub/grub.cfg`. Reboot and then try flashing
+  again.
+
+If above does not resolve the problem the kernel may be compiled with strict
+devmem which prohibits accessing the IOMEM. You should then take different
+Linux system.
+
+### Ubuntu installation
+
+Ubuntu legacy installers have problems to setup graphical mode. When you see an
+error:
+
+``` console
+graphics initialization failled
+Error setting up gfxboot
+boot:_
+```
+
+You need a workaround to proceed with installation. To boo the installer simply
+type `live-install` and press `ENTER`. It will boot to Ubuntu live and the
+installer will be automatically launched.
+
+Version affected: Dasharo Workstation v0.1
+
+If you see blinking yellow LED and black screen after reboot:
```

Comment:
Is this bug possible to fix by some hardware DIY ?

Reply:
@mikebdp2 there is not need to fix, only v0.1 is affected, v0.2 and future version have that fixed.